gcp-logging/src/main/java/io/micronaut/gcp/logging/StackdriverJsonLayout.java

```diff
@@ -54,7 +54,7 @@ public StackdriverJsonLayout() {
         this.includeException = false;
         this.includeTraceId = true;
         this.includeSpanId = true;
-        ObjectMapper mapper = new ObjectMapper();
+        ObjectMapper mapper = ObjectMapper.getDefault();
```

Comment:
use `JsonMapper:createOrDefault` ?

Reply:
I am making this change, but note that the `gcp-logging` module (as well as `gcp-tracing`) are still disabled in settings.gradle.